packages/block-library/src/cover/edit.js

```diff
@@ -604,7 +618,7 @@ function CoverEdit( {
 	const classes = classnames(
 		dimRatioToClass( dimRatio ),
 		{
-			'is-dark-theme': isDark,
+			'is-light': ! isDark,
```

Comment:
`is-light` is preferable, since the current default for the block is dark. We can't update all the blocks that already exist in post content, so the class works better as a modification to a new style rather than a change to the existing rules.